bin/generate-css-utility-classes-docs.js

```diff
@@ -145,17 +144,23 @@ async function generateUtilityClassesDocumentation(
   styleSheet,
   fileWriter = file.writeFile,
 ) {
-  // eslint-disable-next-line no-console
-  console.log('Grouping stylesheet rules by CSS property');
+  if (!process.env.CI) {
+    // eslint-disable-next-line no-console
+    console.log('Grouping stylesheet rules by CSS property');
+  }
+
   const rulesForStorybook = groupCssRulesByCssProperty(styleSheet.cssRules);
 
   for (const [cssProperty, cssRules] of Object.entries(rulesForStorybook)) {
     const storybookContent = generateUtilityClassStories(cssProperty, cssRules);
 
-    // eslint-disable-next-line no-console
-    console.log(
-      `Persisting Storybook stories for CSS utility classes related to the ${cssProperty} property.`,
-    );
+    if (!process.env.CI) {
```

Comment:
Hmm, I wonder if this would be useful enough to extract into a utility function somewhere and then replace the console.log calls with that, something like  this:

```js
function log_unless_ci(message) {
  if (process.env.CI) return;

  console.log(message);
}